src/test/java/org/fundacionjala/coding/ana/TestWord.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.ana;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * word invert when the word is major that 5.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class TestWord {
+    private Word invertWord;
+
+    /**
+     * call to class Word.
+     */
+    @Before
+    public void setUp() {
+        invertWord = new Word();
+    }
+
+    /**
+     * test for invert the word.
+     */
+    @Test
+    public void tesTestWordInvertirCadenaCompareto() {
+        assertEquals("hey wollef sroirraw", invertWord.stringWords("hey fellow warriors"));
+        System.out.println(invertWord.stringWords("hey fellow warriors"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)